website/docs/migrating-from-docker/customizing-docker-compatibility.md

```diff
@@ -0,0 +1,177 @@
+---
+sidebar_position: 2
+title: Customizing Docker compatibility
+description: How to customize the Docker compatibility feature
+keywords: [podman desktop, podman, Docker compatibility, Kubernetes]
+tags: [enable-docker-compatibility, disable-docker-compatibility]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Customizing Docker compatibility - Experimental feature
+
+With Podman Desktop, you can customize the Docker compatibility feature. If you want to run your Docker applications on a Podman engine, you can enable the feature. Otherwise, keep the feature disabled.
+
+**_On macOS_**
```

Comment:
This section should use tabs to show macOS vs Windows vs Linux comparisons (Linux is missing too?)